### PR TITLE
Ravage and Shrike Screech can be selected with a keybind

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

You can now select these abilities with two new keybinds, and then use shift/middle/right click to proc them.
Doesn't change the old keybinds, you can still use them to immediatly proc the abilities

## Why It's Good For The Game

I've always found very strange that these abilities, that can be targeted with a click, can be immediatly used with a keybind like stomp or ignore pain.

I'm not removing that, because some players may prefer it, but i'm adding a new way to use those two abilities. I'll personnaly use it, because i'm missing two many ravage and screech because of the current system

## Changelog
:cl:
qol: Ravage and Shrike Screech can be selected with a new keybind, and then used with shift/middle/right click
/:cl:

### CODE REVIEW

File: code/datums/actions/xeno_action.dm [hunk 1]
```diff
@@ -194,6 +212,7 @@
 	if(can_use_action(FALSE, NONE, TRUE)) // just for selecting
 		action_activate()
 
+
```
Comment (BraveMole) [inline code/datums/actions/xeno_action.dm:215]:
```suggestion
```

File: code/modules/mob/living/carbon/xenomorph/castes/bull/abilities_bull.dm [hunk 2]
```diff
@@ -10,15 +10,6 @@
 	var/new_charge_type = CHARGE_BULL
 
 
-/datum/action/xeno_action/activable/action_activate()
```
Comment (BraveMole) [inline code/modules/mob/living/carbon/xenomorph/castes/bull/abilities_bull.dm:13]: I moved this cause it was at a very random place

**Comment by Mannybrado** [comment]
epic. dont know why it was changed in the first place; they still depend on where you're facing, not where your mouse is

File: code/datums/actions/xeno_action.dm [hunk 3]
```diff
@@ -172,17 +172,35 @@
 
 
 /datum/action/xeno_action/activable
+	///Alternative keybind signal, that will always select the ability, even ignoring keybind flag
+	var/alternate_keybind_signal
```
Comment (SplinterGP) [inline code/datums/actions/xeno_action.dm:176]: better, make the keybind  signal var a list with keybinds with assoc of w hat is their behaviour, would work better i believe and you could add multiple keybinds on future that way

**Review by SplinterGP — CHANGES_REQUESTED** [review]

File: code/datums/actions/xeno_action.dm (hunk 3 above)
Comment (BraveMole) [reply code/datums/actions/xeno_action.dm:176]: Completly out of the scope this PR.

I'm adding two alternates keybind, i'm not refactoring the whole shit.

"you could add multiple keybinds on future that way" I won't, you won't, no one will. And if anyone will, he'll do it himself

**Comment by SplinterGP** [comment]
reviiew this pls

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**